src/components/Artwork.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-function Artwork ({ img, blurLevel, gameEnd }) {
+function Artwork ({ img, blurLevel, gameEnd, hidden }) {
```

Comment:
`hidden` -> `isHidden`

Reply:
I think we better stick to `hidden `to align with the HTML attribute